content/applications/hr/attendances/check_in_check_out.rst

```diff
@@ -0,0 +1,89 @@
+:show-content:
+
+================
+Check in and out
+================
+
+Odoo's *Attendances* application allows users who are logged into the database to check in and out,
+without needing to go into the *Attendances* application, or use a kiosk. For some smaller
+companies, where every employee is also a user, this feature may be useful.
+
+A user can check in and out on the main database dashboard, as well as in any application. In the
+upper right corner of the top main menu, which is always visible regardless of what application the
+user is in, a :guilabel:`🔴 (red circle)` or :guilabel:`🟢 (green circle)` is visible. Click on the
+colored circle to reveal the attendance widget, enabling the user to check in and/or out.
+
+.. image:: check_in_check_out/top-menu.png
+   :align: center
+   :alt: Top right main menu with check in button highlighted.
+
+.. _attendances/check-in:
+
+Check in
+========
+
+If the attendance widget circle is red, this indicates the user is not currently checked in. Click
+the :guilabel:`🔴 (red circle)` and the attendance widget appears, displaying a green
+:guilabel:`Check in` :icon:`fa-sign-in` button.
+
+.. image:: check_in_check_out/check-in.png
+   :align: center
+   :alt: Top right main menu with check in button highlighted.
+
+When the user checks in from the database, the *Attendances* app logs the location details for the
+user, including the IP Address and GPS coordinates.
+
+.. important::
+   For the *Attendances* app to log the location details, the user must allow their computer to
+   access their location information.
+
+If the user has not checked in and out already during the current work day, this button is the only
+visible item in the widget. If the user has previously checked in and out, a :guilabel:`Total today`
+field appears above the button, and the total amount of time that has been logged for the day
+appears beneath that field, in an :guilabel:`HH:MM` (hours:minutes) format.
+
+Click the :guilabel:`Check in` :icon:`fa-sign-in` button to check in. The :guilabel:`🔴 (red
+circle)` in the top menu changes to green, and the widget changes appearance as well. The widget
+updates to reflect that the user has checked in, by changing the green :guilabel:`Check in`
+:icon:`fa-sign-in` button to a yellow :guilabel:`Check out` :icon:`fa-sign-out` button.
+
+Click anywhere on the screen to close the attendance widget.
+
+Check out
+=========
+
+If the user is checking out for the first time, :guilabel:`Since HH:MM (AM/PM)` appears at the top
+of the widget, with the time the user checked in populating the time field. Beneath that line,
+:guilabel:`HH:MM` is displayed, indicating the hours and minutes that have elapsed since checking
+in. As time passes, this value is updated to reflect the hours and minutes that have passed since
```

Comment:
```suggestion
of the widget, with the time the user checked in populating the time field. Beneath that line,
the hours and minutes that have elapsed since checking
in are displayed, in a :guilabel:`HH:MM`  format. As time passes, this value is updated to reflect the hours and minutes that have passed since
```